src/App.tsx

```diff
@@ -6,18 +6,22 @@ import Contact from "./components/templates/Contact";
 import { VideoListPage } from "./components/templates/VideoListPage";
 import SeriesPage from "./components/templates/SeriesPage";
 import AnimationPage from "./components/templates/AnimationPage";
+import { Header } from "./components/molecules/headers/Header";
 
 function App() {
   return (
-    <BrowserRouter>
-      <Routes>
-        <Route path="/" element={<Home />} />
-        <Route path="/contact" element={<Contact />} />
-        <Route path="/video-list-page" element={<VideoListPage />} />
-        <Route path="/series" element={<SeriesPage />} />
-        <Route path="/animation" element={<AnimationPage />} />
-      </Routes>
-    </BrowserRouter>
+    <div>
```

Comment:
❓ Why wrap it all in a div?

Reply:
Without the `div` wrapping everything inside of the `return` the file errors with this message:
"JSX expressions must have one parent element."

Reply:
Ah.  Ya. that's true.  Fun shortcut to indicate that you're wrapping but don't really care about the element it is:

```typescript
<>
  <Header />
  <BrowserRouter>
    ...
  </BrowserRouter>
</>
```